README.md

```diff
@@ -652,6 +653,51 @@ The supported Picture in Picture events on `PlayerView` are:
 
 Check [`events.ts`](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/src/components/PlayerView/events.ts) for more information about them.
 
+### Customize HTML UI (Android and iOS only)
```

Comment:
@stonko1994 please review the README changes here if my descriptions for custom HTML Player UI make sense.